src/test/kotlin/calculator/OperandTest.kt

```diff
@@ -0,0 +1,22 @@
+package calculator
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+internal class OperandTest {
+    @ParameterizedTest
+    @ValueSource(strings = ["+", "a", "ㅎㅎ"])
+    fun `숫자가 아닌 입력값이 있는 경우 예외`(input: String) {
+        assertThrows<RuntimeException> { Operand.from(input) }
+            .also { Assertions.assertThat(it.message).isEqualTo("피연산자는 숫자만 입력하실 수 있습니다.") }
```

Comment:
여기서는 굳이 scope function을 사용하지 않아도 괜찮을 것 같아요!
```suggestion
        assertThrows<RuntimeException> { Operand.from(input) }
            .shouldHaveMessage("피연산자는 숫자만 입력하실 수 있습니다.")
```

Reply:
오 이런게 있는지 몰랐네요! 👍